tsconfig.json

```diff
@@ -3,7 +3,9 @@
     "moduleResolution": "node",
     "noEmit": true,
     "resolveJsonModule": true,
-    "noImplicitAny": true
+    "noImplicitAny": true,
```

Comment:
`"noImplicitAny": true` can be dropped because `"strict": true` already includes it.